scale-tests/setup_streaming.py

```diff
@@ -92,10 +93,14 @@ def install_package(package_name: str, service_prefix: str, index: int, service_
     return {"package_name": package_name, **service_options}
 
 
+def _supports_multiple_clusters(service_name: str) -> bool:
+    return service_name in SUPPORTED_MULTIPLE_CLUSTER_SERVICES
```

Comment:
Since the global is not used elsewhere, let's just use a local variable here as it makes it easier to read without having to refer to other sections in the code.

Reply:
In these situations I generally prefer not defining variables inside functions (they can get "lost" within the code easier), and instead either pass parameters or use constants. If you feel strongly about it I can change it though.

Reply:
https://github.com/mesosphere/spark-build/pull/336/commits/5fbeabe1e634ffc03d256b4360ab9d6740b57a4e